<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="QWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::WizardStyle::ModernStyle</enum>
  </property>
  <property name="options">
   <set>QWizard::WizardOption::HelpButtonOnRight</set>
  </property>
  <widget class="QWizardPage" name="wizardPage1_launch">
   <widget class="QPushButton" name="pushButton_launch">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage2_model">
   <widget class="QPushButton" name="pushButton_load">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::GoDown"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_project">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>.femprj</string>
    </property>
    <property name="buddy">
     <cstring>plainTextEdit_prj</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label_model">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>220</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>model</string>
    </property>
    <property name="buddy">
     <cstring>plainTextEdit_model</cstring>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_prj">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>&lt;.femprj path&gt;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_model">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>&lt;analysis model name&gt;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="markdown">
     <string>.femprj ファイルを開いてください。

目的の解析モデルを開き、Load を押してください。

</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yu Gothic UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;.femprj ファイルを開いてください。&lt;/p&gt;
&lt;p style=&quot; margin-top:6px; margin-bottom:6px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;目的の解析モデルを開き、Load を押してください。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage3_param">
   <widget class="QPushButton" name="pushButton_load_prm">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>85</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::GoDown"/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_prm">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>9</y>
      <width>585</width>
      <height>291</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::AllEditTriggers</set>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>328</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yu Gothic UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;最適化する変数を選んでください。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;初期値、上限、下限を指定してください。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage4_obj">
   <widget class="QTextEdit" name="textEdit_3">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>309</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yu Gothic UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;最適化する目的関数を選んでください。&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;※ Femtet のパラメトリック解析ダイアログで目的関数を定義してください。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_obj">
    <property name="geometry">
     <rect>
      <x>81</x>
      <y>0</y>
      <width>585</width>
      <height>291</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::AllEditTriggers</set>
    </property>
    <property name="verticalScrollMode">
     <enum>QAbstractItemView::ScrollMode::ScrollPerItem</enum>
    </property>
    <property name="horizontalScrollMode">
     <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load_obj">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>76</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::GoDown"/>
    </property>
   </widget>
  </widget>
  <widget class="QWizardPage" name="wizardPage9_verify">
   <widget class="QTextEdit" name="textEdit_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Yu Gothic UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;よろしいですか？&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTreeView" name="treeView">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>421</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_launch</sender>
   <signal>clicked()</signal>
   <receiver>Wizard</receiver>
   <slot>connect_process()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked()</signal>
   <receiver>Wizard</receiver>
   <slot>load_model()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>-20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load_prm</sender>
   <signal>clicked()</signal>
   <receiver>Wizard</receiver>
   <slot>load_prm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>-12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load_obj</sender>
   <signal>clicked()</signal>
   <receiver>Wizard</receiver>
   <slot>load_obj()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>process_loaded()</signal>
  <signal>check_femtet_alive()</signal>
  <slot>connect_process()</slot>
  <slot>load_model()</slot>
  <slot>load_prm()</slot>
  <slot>load_obj()</slot>
  <slot>enable_next_button()</slot>
 </slots>
</ui>
